src/ngraph/pass/convert_elimination.cpp

```diff
@@ -0,0 +1,93 @@
+//*****************************************************************************
+// Copyright 2017-2020 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+#include <functional>
+#include <memory>
+#include <typeindex>
+#include <typeinfo>
+#include <unordered_map>
+
+#include "convert_elimination.hpp"
+#include "ngraph/graph_util.hpp"
+#include "ngraph/log.hpp"
+#include "ngraph/op/convert.hpp"
+#include "ngraph/opsets/opset3.hpp"
+
+using namespace std;
+using namespace ngraph;
+
+#define TI(x) x::type_info
+
+//************************
+// Eliminates 2 types of spines with converts:
+//
+// 1. [ op1--> convert2(type i64->i32)--> ReduceMin--> convert1(type i32->i64)--> op2 ] to
+//                    [ op1--> ReduceMin--> op2 ]
+// 2. [ op3--> convert1-->  NonZero--> op4 ] to
+//                    [ op3--> NonZero--> op4 ]
+//
+//*************************
+static bool eliminate_convert(const std::shared_ptr<Node>& node)
```

Comment:
make this a private static member function of ConvertElimination to reduce its scope.

Reply:
done